PowerEditor/src/functionList.xml

```diff
@@ -176,6 +176,94 @@
 							|	(?s:\x27(?:[^\x27\x5C]|\x5C.)*\x27)             # String Literal - Single Quoted
 							"
 			>
+				<classRange
+					mainExpr    ="^[\t\x20]*(class|struct)[\t\x20]+\w+\s*(final)?\s*(:\s*(AAAAAAAAAAAAAAAAAAAAAAAAAAAA99)\s+\w+\s*)?\{"

```

Comment:
`(AAAAAAAAAAAAAAAAAAAAAAAAAAAA99)` does not seem right to me.
Furthermore, you applied the changes to the wrong parser i.e. the `C` instead of `C++` parser.
Did you read the guidelines for contributing?